netket/models/equivariant.py

```diff
@@ -139,6 +138,6 @@ def __call__(self, x_in):
 
         if not self.output_activation == None:
             x = self.output_activation(x)
-        x = jnp.sum(x, axis=-1)
+        x = jnp.sum(x, axis=-1) / np.sqrt(x.shape[-1])
```

Comment:
why `sqrt`? can you put a comment explaining the choice of normalisation?

Reply:
I'll add a comment.

This is basically just having variance scaling in the final layer. This makes sure the GCNN always returns outputs with unit normal distribution at initialization